src/porepy/numerics/fv/upwind.py

```diff
@@ -410,7 +410,7 @@ def discretize(
         # The mortars always points from upper to lower, so we don't flip any signs. The
         # mapping will be non-zero also for faces not adjacent to the mortar grid,
         # however, we wil hit it with mortar projections, thus kill those elements
-        inv_trace_h = np.abs(pp.fvutils.scalar_divergence(sd_primary))
+        inv_trace_h = sd_primary.divergence(n=1)
```

Comment:
There's an absolute value missing from this trace operator.

Reply:
This is also a case where a real trace operator is needed. Taking the absolute value of the divergence happens to be functionally equivalent to the trace, but it is conceptually wrong.

Reply:
Are you suggesting a different implementation?